seed/tests/test_utils_ubid.py

```diff
@@ -162,20 +162,23 @@ def test_decode_ulids_does_nothing_if_no_ULID_provided(self):
     def test_decode_ubids_is_successful_when_v2_format_C_NW_SE_UBID_provided(self):
         property_details = self.property_state_factory.get_details()
         property_details['organization_id'] = self.org.id
-        property_details['ubid'] = '849VQJH6+95J-849VQJH5+VGW-849VQJG6+XV8'
+        # Old format (v2) is not longer supported
```

Comment:
v3 is tested already, so this particular test could be removed. Not a big deal either way though.